src/zeit/content/cp/blocks/automatic.py

```diff
@@ -87,6 +87,24 @@ def layout(self, layout):
     def change_layout(self, layout):
         self.temporary_layout = layout
 
+    def materialize(self):
+        # Most of this code is quite similar to Area._materialize_auto_blocks,
+        # but not similar enough to be extracted.
+        area = self.__parent__
+
+        block_id = self.__name__
+        rendered = zeit.content.cp.interfaces.IRenderedArea(area)
+        for block in rendered.values():
+            if block.__name__ == block_id:
+                auto_filled = block
+                break
+
+        order = area.keys()
+        del area[block_id]
+        materialized = area.create_item('teaser')
+        area.copy_teaserlist_attributes(auto_filled, materialized)
```

Comment:
Die Methode gehört eigentlich nach `zeit.content.cp.blocks.teaser.TeaserBlock`. Dann ist die Auswahl vom Interface nah an der Implementierung (siehe `TEASERBLOCK_FIELDS` in `zeit.content.cp.area`) und alle TeaserBlöcke können bereits Attribute von einem anderen runter kopieren. Verwendung wäre dann etwa `materialized.copy_attributes_from(auto_filled)`.

Ist mir bereits beim Umbau aufgefallen, dass es vermutlich eine Funktion ist, die generell verfügbar sein sollte. :)